MYR_rails/config/routes.rb~

```diff
@@ -1,193 +0,0 @@
-Rails.application.routes.draw do
```

Comment:
This was a copy of routes.rb that already exists.

Reply:
Ah, thanks. The `~` files are backups made by vim, I think. They're not usually checked into version control.